### ISSUE TITLE
Eror in log file

### ISSUE THREAD

**Issue by daves00** [jellyfin/jellyfin#4780]
**Describe the bug**

Some metadata isn't being populated. The only error is see in the logs is below, perhaps related, or not?

**System (please complete the following information):**
 - OS:Ubuntu 18.04
 - Virtualization: None
 - Clients: Windows Firefox
 - Browser: Firefox 83
 - Jellyfin Version:10.6.4
 - Playback: Direct Play
 - Installed Plugins: none, 
 - Reverse Proxy: none
 - Base URL: default  (ip addr:8096/web/index.html)
 - Networking: Simple home Lan using 192 addresses
 - Storage: local

**To Reproduce**
Run scan libraries

**Expected behavior**

**Logs**

[2020-12-12 17:20:33.390 +00:00] [INF] Creating image collage and saving to "/var/cache/jellyfin/temp/4f043cd2f9ed444f82bab85e9c0c6f71.png"
[2020-12-12 17:20:33.674 +00:00] [INF] Completed creation of image collage and saved to "/var/cache/jellyfin/temp/4f043cd2f9ed444f82bab85e9c0c6f71.png"
[2020-12-12 17:20:33.742 +00:00] [INF] OnRefreshStart "f669abac61912a0bba0b0399b2181f3b"
[2020-12-12 17:20:37.110 +00:00] [ERR] Error in "MusicAlbumResolver" resolving "/mnt/mediastore/music/Jeff Beck"
System.IO.IOException: Input/output error
   at System.IO.FileStatus.EnsureStatInitialized(ReadOnlySpan`1 path, Boolean continueOnError)
   at System.IO.FileSystemInfo.Init(FileStatus& fileStatus)
   at System.IO.FileSystemInfo.Create(String fullPath, String fileName, FileStatus& fileStatus)
   at System.IO.Enumeration.FileSystemEnumerableFactory.<>c.<FileInfos>b__6_0(FileSystemEntry& entry)
   at System.IO.Enumeration.FileSystemEnumerator`1.MoveNext()
   at System.Linq.Enumerable.SelectEnumerableIterator`2.MoveNext()
   at System.Collections.Generic.LargeArrayBuilder`1.AddRange(IEnumerable`1 items)
   at System.Collections.Generic.SparseArrayBuilder`1.ReserveOrAdd(IEnumerable`1 items)
   at System.Linq.Enumerable.Concat2Iterator`1.ToArray()
   at MediaBrowser.Controller.Providers.DirectoryService.GetFileSystemEntries(String path)
   at Emby.Server.Implementations.Library.Resolvers.Audio.MusicAlbumResolver.ContainsMusic(IEnumerable`1 list, Boolean allowSubfolders, IDirectoryService directoryService, ILogger`1 logger, IFileSystem fileSystem, ILibraryManager libraryManager)
   at Emby.Server.Implementations.Library.Resolvers.Audio.MusicAlbumResolver.IsMusicAlbum(ItemResolveArgs args)
   at Emby.Server.Implementations.Library.Resolvers.Audio.MusicAlbumResolver.Resolve(ItemResolveArgs args)
   at Emby.Server.Implementations.Library.Resolvers.ItemResolver`1.MediaBrowser.Controller.Resolvers.IItemResolver.ResolvePath(ItemResolveArgs args)
   at Emby.Server.Implementations.Library.LibraryManager.Resolve(ItemResolveArgs args, IItemResolver resolver)
[2020-12-12 17:20:37.142 +00:00] [ERR] Error resolving path "/mnt/mediastore/music/Jeff Buckley"
System.IO.IOException: Input/output error
   at System.IO.FileStatus.EnsureStatInitialized(ReadOnlySpan`1 path, Boolean continueOnError)
   at System.IO.FileSystemInfo.Init(FileStatus& fileStatus)
   at System.IO.FileSystemInfo.Create(String fullPath, String fileName, FileStatus& fileStatus)
   at System.IO.Enumeration.FileSystemEnumerableFactory.<>c.<FileInfos>b__6_0(FileSystemEntry& entry)
   at System.IO.Enumeration.FileSystemEnumerator`1.MoveNext()
   at System.Linq.Enumerable.SelectEnumerableIterator`2.MoveNext()
   at System.Collections.Generic.LargeArrayBuilder`1.AddRange(IEnumerable`1 items)
   at System.Collections.Generic.SparseArrayBuilder`1.ReserveOrAdd(IEnumerable`1 items)
   at System.Linq.Enumerable.Concat2Iterator`1.ToArray()
   at MediaBrowser.Controller.Providers.DirectoryService.GetFileSystemEntries(String path)
   at MediaBrowser.Controller.IO.FileData.GetFilteredFileSystemEntries(IDirectoryService directoryService, String path, IFileSystem fileSystem, IServerApplicationHost appHost, ILogger logger, ItemResolveArgs args, Int32 flattenFolderDepth, Boolean resolveShortcuts)
   at Emby.Server.Implementations.Library.LibraryManager.ResolvePath(FileSystemMetadata fileInfo, IDirectoryService directoryService, IItemResolver[] resolvers, Folder parent, String collectionType, LibraryOptions libraryOptions)
   at Emby.Server.Implementations.Library.LibraryManager.<>c__DisplayClass84_0.<ResolveFileList>b__0(FileSystemMetadata f)

**Comment by Artiume** [comment]
```
[2020-12-12 17:20:33.742 +00:00] [INF] OnRefreshStart "f669abac61912a0bba0b0399b2181f3b"
[2020-12-12 17:20:37.110 +00:00] [ERR] Error in "MusicAlbumResolver" resolving "/mnt/mediastore/music/Jeff Beck"
System.IO.IOException: Input/output error
at System.IO.FileStatus.EnsureStatInitialized(ReadOnlySpan1 path, Boolean continueOnError) at System.IO.FileSystemInfo.Init(FileStatus& fileStatus) at System.IO.FileSystemInfo.Create(String fullPath, String fileName, FileStatus& fileStatus) at System.IO.Enumeration.FileSystemEnumerableFactory.<>c.<FileInfos>b__6_0(FileSystemEntry& entry) at 
```
what's the permissions of /mnt/mediastore/music/Jeff Beck

**Comment by daves00** [comment]
On the directory - rwx rwx r x

owner and group and perms are the same as all other folders within the music library

**Comment by Artiume** [comment]
I notice that both Jeff Beck and Jeff Buckley have spaces in the names, could that be involved? I/O errors are typically not a Jellyfin error but a filesystem/file error

**Comment by daves00** [comment]
This has helped me determine the issue. I tried to move the two folders out of the way, and the directories empty but won't delete, giving an I/O error when using the linux command line. Tried an ls on the folder and there are some phantom entries in the directory for the cover art. A bit of digging, btrfs scrub, on my raid 1 pair and 2 uncorrectable errors - so clearly this is an issue at the file system level, so this issue can be closed. Thank you for the reponsses which - highlighting this is likely a filesystem issue steered me to dig at this in the right way! Now to try and clear these errors from the raid array without wiping and rebuilding (to much hassle!)

Thanks All
